lib/harvesting/client.rb

```diff
@@ -57,6 +57,16 @@ def invoices
       end
     end
 
+    def user_assignments(project = nil, opts = {})
+      path = project.nil? ? "user_assignments" : "projects/#{project.id}/user_assignments"
+      Harvesting::Models::ProjectUserAssignments.new(get(path, opts), opts, client: self)
+    end
```

Comment:
@mscottford I think it would be easier to use if you divided this method in two: 

```
# project.rb

def user_assignments
  harvest_client.user_assignments(project_id: self.id)
end
```

and in here it would be a simpler method: 

```
def user_assignments(opts = {})
  path = opts[project_id].nil? ? "user_assignments" : "projects/#{opts[:project_id]}/user_assignments"
  Harvesting::Models::ProjectUserAssignments.new(get(path, opts), opts, harvest_client: self)
end
```

Reply:
Done.